src/citrine/resources/material_run.py

```diff
@@ -108,7 +108,7 @@ def get_type(cls) -> Type[MaterialRun]:
         """Return the resource type in the collection."""
         return MaterialRun
 
-    def get_history(self, scope: Optional[str],
+    def get_history(self, *, scope: Optional[str],
```

Comment:
wasn't sure about this one

Reply:
If the only argument were `id` then I'd support not requiring the keyword. But since `scope`, which is deprecated, was the first argument, if we put `id` first and didn't require a keyword then we'd create unexpected failures. So I think the way you've done it here is what we have to do